modules/shortcodes/dailymotion.php

```diff
@@ -201,7 +201,7 @@ function dailymotion_shortcode( $atts ) {
 		}
 
 		if ( array_key_exists( 'user', $atts ) && $user = preg_replace( '/[^-a-z0-9_]/i', '', $atts['user'] ) ) {
-			$output .= '<br /><em>' . __('Uploaded by') . ' <a href="' . esc_url( 'http://www.dailymotion.com/' . $user ) . '" target="_blank">' . esc_html( $user ) . '</a></em>';
+			$output .= '<br /><em>' . esc_html__( 'Uploaded by', 'jetpack' ) . ' <a href="' . esc_url( 'http://www.dailymotion.com/' . $user ) . '" target="_blank">' . esc_html( $user ) . '</a></em>';
```

Comment:
These translatable strings with links within them are always tricky. The best approach for translation is to keep variables (like the user name in this case) in context since the syntax may vary in different languages. This could be more contextual and understandable by doing
```php
/* translators: %s is a Dailymotion user name */
$output .= '<br /><em>' . sprintf( __( 'Uploaded by %s', 'jetpack' ), '<a href="' . esc_url( 'http://www.dailymotion.com/' . $user ) . '" target="_blank">' . esc_html( $user ) . '</a>' ) . '</em>';
```
the caveat of this is that it ends up open to ill-intentioned HTML strings in the translation file by using `__`. 
So, the final update would be to add `wp_kses` to allow only the `<a>` element with `href` and `target` attributes:
```php
/* translators: %s is a Dailymotion user name */
$output .= '<br /><em>' . wp_kses( sprintf( __( 'Uploaded by %s', 'jetpack' ), '<a href="' . esc_url( 'http://www.dailymotion.com/' . $user ) . '" target="_blank">' . esc_html( $user ) . '</a>' ), array( 'a' => array( 'href' => true, 'target' => true ) ) ) . '</em>';
```